src/components/charts/SwipeChart.tsx

```diff
@@ -48,31 +34,40 @@ import { useHandler } from '../../hooks/useHandler'
 import { formatDate } from '../../locales/intl'
 import { lstrings } from '../../locales/strings'
 import { getCoingeckoFiat } from '../../selectors/SettingsSelectors'
+import { useSceneScrollContext } from '../../state/SceneScrollState'
 import { useSelector } from '../../types/reactRedux'
 import { snooze } from '../../util/utils'
 import { MinimalButton } from '../buttons/MinimalButton'
 import { FillLoader } from '../progress-indicators/FillLoader'
-import { type Theme, useTheme } from '../services/ThemeContext'
-import { ReText } from '../text/ReText'
+import { cacheStyles, type Theme, useTheme } from '../services/ThemeContext'
 import { EdgeText } from '../themed/EdgeText'
 
+// Timespans supported
 type Timespan = 'year' | 'month' | 'week' | 'day' | 'hour'
+
+// CoinGecko API data types
 type CoinGeckoDataPair = number[]
+interface CoinGeckoMarketChartRange {
+  prices: CoinGeckoDataPair[]
+  market_caps: CoinGeckoDataPair[]
+  total_volumes: CoinGeckoDataPair[]
+}
 
+// Component props
 interface Props {
   /** The asset's 'id' as defined by CoinGecko */
+  // This rule doesn't reliably track TS interfaces on fn components...
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
The problem is that you have `export const SwipeChart: React.FC<Props> = (params: Props) => {`, which should instead be `export const SwipeChart: React.FC<Props> = props => {`. Notice the variable rename, and the type removal. With this change in place, ESLint is fine.